composer/datasets/mnist.py

```diff
@@ -54,7 +54,7 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHpara
 class MNISTWebDatasetHparams(WebDatasetHparams):
     """Defines an instance of the MNIST WebDataset for image classification.
 
-    Parameters:
+    Args:
```

Comment:
References to `DataSpec` in the "Returns" section for the `initialize_object` method aren't resolving.